financialaid/constants.py

```diff
@@ -52,11 +52,13 @@ class FinancialAidStatus:
         SKIPPED: "Skipped",
     }
 
-
-CURRENCY_EXCHANGE_RATE_API_REQUEST_URL = "{url}latest.json?app_id={app_id}".format(
-    url=settings.OPEN_EXCHANGE_RATES_URL,
-    app_id=quote_plus(settings.OPEN_EXCHANGE_RATES_APP_ID)
-)
+if settings.OPEN_EXCHANGE_RATES_URL and settings.OPEN_EXCHANGE_RATES_APP_ID:
```

Comment:
Frankly, I don't think this should be a variable at all -- it's not like we'll be able to just swap out which API we use. That's a larger change, though, that may not be in scope of this pull request. I've set a default for this variable in the `settings.py` file.